parsers/FR.py

```diff
@@ -216,24 +178,14 @@ def fetch_consumption(
 
     # reindex df_consumption to get 1/2 hourly values
     df_consumption_reindexed = reindex_data(df_consumption)
-    datapoints = []
+    consumption_list = TotalConsumptionList(logger)
     for row in df_consumption_reindexed.itertuples():
-        datapoints.append(
-            {
-                "zoneKey": zone_key,
-                "datetime": arrow.get(row.date_heure)
-                .replace(tzinfo="Europe/Paris")
-                .datetime,
-                "consumption": row.consommation,
-                "source": "opendata.reseaux-energies.fr",
-            }
+        consumption_list.append(
+            zoneKey=zone_key,
+            consumption=row.consommation,
+            datetime=row.Index.to_pydatetime(),
+            source=SOURCE,
+            sourceType=EventSourceType.measured,
```

Comment:
Technically this don't need to be included as it's the default. But it don't hurt either.